tests/unit/conftest.py

```diff
@@ -429,3 +429,11 @@ def mock_charm_fixture():
     mock_charm = unittest.mock.MagicMock(spec=CharmBase)
     mock_charm.app.planned_units.return_value = 1
     return mock_charm
+
+
+@pytest.fixture(scope="function", name="patch_os_environ")
```

Comment:
What about just passing the `autouse` argument so it doesn't have to be manually applied to every test? Or is it not needed on some tests?